### PR TITLE
Add callback exceptions management

### PR DESCRIPTION
**Pull request by ThibaudDauce**

### CODE REVIEW

File: src/main/java/com/cosmian/jna/findex/FindexCallbackException.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import java.util.ArrayList;
+import java.util.List;
+
+import com.cosmian.utils.CloudproofException;
+
+public class FindexCallbackException {
+
+    private final static int CALLBACK_ERROR_CODE_WHEN_THROWING = 42;
+
+    public static List<FindexCallbackException> exceptions = new ArrayList<>();
```
Comment (Manuthor) [inline src/main/java/com/cosmian/jna/findex/FindexCallbackException.java:14]: should we purge the content of this list at one moment?

Comment (Manuthor) [reply src/main/java/com/cosmian/jna/findex/FindexCallbackException.java:14]: the variable does not look safe for multithreaded, what do you think?

Comment (ThibaudDauce) [reply src/main/java/com/cosmian/jna/findex/FindexCallbackException.java:14]: I don't know how Java is dealing with multithreading… I was hoping it was thread safe by default?

Comment (ThibaudDauce) [reply src/main/java/com/cosmian/jna/findex/FindexCallbackException.java:14]: Purge done in 0119642

**Review by Manuthor — APPROVED** [review]

**Pull request merged**